apps/web/src/pages/templates/workflow/DigestMetadata.tsx

```diff
@@ -1,213 +1,208 @@
+import { TypeSegmented } from './digest/TypeSegment';
+import { WillBeSentHeader } from './digest/WillBeSentHeader';
+import { RegularInfo } from './digest/icons/RegularInfo';
+import { TimedDigestMetadata } from './TimedDigestMetadata';
+import { RegularDigestMetadata } from './RegularDigestMetadata';
 
-const StyledSwitch = styled(Switch)`
-  max-width: 100% !important;
-  margin-top: 15px;
-`;
+export const DigestMetadata = ({ index, readonly }: { index: number; readonly: boolean }) => {
+  const { control, watch } = useFormContext();
 
-export const DigestMetadata = ({ control, index }) => {
-  const { readonly } = useEnvController();
-  const {
-    formState: { errors, isSubmitted, isDirty },
-    watch,
-    trigger,
-  } = useFormContext();
+  const { colorScheme } = useMantineColorScheme();
 
-  const type = watch(`steps.${index}.metadata.type`);
-  const showErrors = isSubmitted && errors?.steps;
+  const type = watch(`steps.${index}.digestMetadata.type`);
+  const digestKey = watch(`steps.${index}.digestMetadata.digestKey`);
+  const isEnterprise = process.env.REACT_APP_DOCKER_HOSTED_ENV !== 'true';
 
   return (
-    <>
-      <div
-        style={{
-          marginBottom: '15px',
-        }}
-      >
-        <Controller
-          control={control}
-          defaultValue={DigestTypeEnum.REGULAR}
-          name={`steps.${index}.metadata.type`}
-          render={({ field }) => {
-            return (
-              <SegmentedControl
-                {...field}
-                sx={{
-                  maxWidth: '100% !important',
+    <div data-test-id="digest-step-settings-interval">
+      <Accordion>
+        <Tooltip
+          position="left"
+          width={227}
+          multiline
+          label="Types of events that will be aggregated from the previous digest to the time it will be sent"
+        >
+          <Accordion.Item value="events-selection" data-test-id="digest-events-selection-options">
+            <Accordion.Control>
+              <Group>
+                <Bell color={colors.B60} />
+                <div>
+                  <div>
+                    <b
+                      style={{
+                        color: colorScheme === 'dark' ? colors.B80 : colors.B40,
+                      }}
+                    >
+                      All events
+                    </b>
+                  </div>
+                  <div>since previous digest</div>
+                </div>
+              </Group>
+            </Accordion.Control>
+            <Accordion.Panel>
+              <Select mt={-5} mb={-5} data={[{ value: 'all', label: 'All events' }]} value={'all'} />
+            </Accordion.Panel>
+          </Accordion.Item>
+        </Tooltip>
+        <Tooltip
+          position="left"
+          width={227}
+          multiline
+          label={
+            <>
+              Events aggregated by subscriber_ID by default, this can’t be changed. You may add additional aggregations
```

Comment:
```suggestion
              Events aggregated by subscriber_ID by default, this can’t be changed. You may add additional aggregations
```
```suggestion
              Events aggregated by subscriberId by default, this can’t be changed. You may add additional aggregations
```